cal.py

```
@@ -0,0 +1,10 @@
+
+from constants import *
```

Ruff (0.8.2)

`from constants import *` used; unable to detect undefined names

(F403)